doc/mgr/telemetry.rst

```diff
@@ -64,53 +64,151 @@ the way Ceph is used.
 
+If telemetry is off you can preview a sample device report with::
+
+  ceph telemetry preview-device
+
 Please note: In order to generate the device report we use Smartmontools
 version 7.0 and up, which supports JSON output. 
 If you have any concerns about privacy with regard to the information included in
 this report, please contact the Ceph developers.
 
-Channels
---------
+In case you prefer to have a single output of both reports, and telemetry is on, use::
 
-Individual channels can be enabled or disabled with::
+  ceph telemetry show-all
 
-  ceph config set mgr mgr/telemetry/channel_ident false
-  ceph config set mgr mgr/telemetry/channel_basic false
-  ceph config set mgr mgr/telemetry/channel_crash false
-  ceph config set mgr mgr/telemetry/channel_device false
-  ceph config set mgr mgr/telemetry/channel_perf false
-  ceph telemetry show
-  ceph telemetry show-device
+Otherwise use::
 
-Enabling Telemetry
-------------------
+  ceph telemetry preview-all
 
-To allow the *telemetry* module to start sharing data::
+**Sample report by channel**
 
-  ceph telemetry on
+When telemetry is on you can see what data is reported by channel with::
 
-Please note: Telemetry data is licensed under the Community Data License
-Agreement - Sharing - Version 1.0 (https://cdla.io/sharing-1-0/). Hence,
-telemetry module can be enabled only after you add '--license sharing-1-0' to
-the 'ceph telemetry on' command.
+  ceph telemetry show <channel_name>
 
-Telemetry can be disabled at any time with::
+Please note: If telemetry is on, and <channel_name> is disabled, the command
+above will output a sample report by that channel, according to the collections
+the user is enrolled to. However this data is not reported, since the channel
+is disabled.
 
-  ceph telemetry off
+If telemetry is off you can preview a sample report by channel with::
+
+  ceph telemetry preview <channel_name>
+
+Collections
+-----------
+
+Collections represent what we collect by channel.
+
+List all collections with::
+
+  ceph telemetry collection ls
+
+  NAME                  ENROLLED    STATUS    DEFAULT    DESC
+  basic_base            FALSE       OFF       ON         Basic information about the cluster (capacity, number and type of daemons, version, etc.)
+  basic_mds_metadata    FALSE       OFF       ON         MDS metadata
+  crash_base            FALSE       OFF       ON         Information about daemon crashes (daemon type and version, backtrace, etc.)
+  device_base           FALSE       OFF       ON         Information about device health metrics
+  ident_base            FALSE       OFF       OFF        User-provided identifying information about the cluster
+  perf_perf             FALSE       OFF       OFF        Information about performance counters of the cluster
+
+Where:
+
+**NAME**: Collection name; prefix indicates the channel the collection belongs to.
+
+**ENROLLED**: Signifies the collections that were available in the module when
```

Comment:
this sounds like an internal detail - does a user need to know what previous collections there were? It seems like that's what the preview command does.

Perhaps presenting it to the user as 'reported' or not would be clearer, instead of having an 'enrolled' column. IIUC there's no way to enable a specific collection, you're always enabling the current version of a whole channel. 'enrolled' is confusing without the context of this doc.